### DISCUSSION TITLE
Could you give me your hand on how to turn off the video microphone?

### DISCUSSION THREAD

**Discussion by nthnhung23** [HaishinKit/HaishinKit.swift#1183] [Q&A] [answered]
I want to turn off or turn on the microphone. I have tried to find the way to do this, but I can't. Could you give me your hand on how to turn off the video microphone? Thanks.

**Comment by shogo4405** [comment, marked as answer]
Please set `stream.hasAudio = false`.

**Reply by nthnhung23** [reply]
Many thanks @shogo4405. I'm trying it now.

**Comment by nthnhung23** [comment]
hi @shogo4405 , 
I would like to capture the MTHKView screen, but after I have used UIGraphicsGetImageFromCurrentImageContext. I got a black MTHKView. Could you give me a hand?

**Reply by shogo4405** [reply]
GPU-based views cannot be captured with UIGraphicsGetImageFromCurrentImageContext. This is due to a limitation in iOS.

**Comment by nthnhung23** [comment]
Hi @shogo4405 
I want to apply overlay video livestream. Have any example or tutorial about it? 
Thanks you

**Reply by nthnhung23** [reply]
@shogo4405 please help me !!!
